.devcontainer/Dockerfile

```diff
@@ -110,6 +105,11 @@ RUN mkdir -p $(include-what-you-use -print-resource-dir 2>/dev/null)
 RUN ln -s $(readlink -f /usr/lib/clang/${LLVM_VER}/include) \
     $(include-what-you-use -print-resource-dir 2>/dev/null)/include
 
+# Install editors
+RUN apt-get update -qq && export DEBIAN_FRONTEND=noninteractive && \
+    apt-get install -y --no-install-recommends \
+        neovim emacs nano
```

Comment:
I really prefer this being earlier, so that no one has to wait for this stuff to download after making other changes to the dockerfile. The whole point of installing all of these is so that no one has to worry about having to install their favorite editor manually. If the end user does not like this selection of editors, they are free to remove the ones they don't like as they see fit. I do not expect an end user to want to change these often, if at all.

Reply:
Well in my use case I use VSCode I don't need a separate editor at all. Why should I download all those editors at all? OK should I put them before compilers installation?

Reply:
You are welcome to remove these editors in your own dockerfile.

Regardless of how you wish to use the dockerfile, this file will be used by others in other ways. Some users will choose to use this dockerfile interactively from a terminal, and will need an editor.

Reply:
Since this is the least frequent I will move it on top